src/h5web/toolbar/controls/DomainSlider/DomainSlider.tsx

```diff
@@ -0,0 +1,107 @@
+import type { ReactElement } from 'react';
+import ReactSlider from 'react-slider';
+import styles from './DomainSlider.module.css';
+import type { CustomDomain, Domain } from '../../../vis-packs/core/models';
+import { extendDomain } from '../../../vis-packs/core/utils';
+import ToggleBtn from '../ToggleBtn';
+import { FiZap } from 'react-icons/fi';
+import { useKey, useToggle } from 'react-use';
+import DomainTooltip from './DomainTooltip';
+import { format } from 'd3-format';
+
+const EXTEND_FACTOR = 0.2;
+const NB_DECIMALS = 1;
+const TOOLTIP_ID = 'domain-tooltip';
+
+const formatThumb = format(`.1f`);
+
+function getAutoscaleLabel(isAutoMin: boolean, isAutoMax: boolean): string {
+  if (isAutoMin && !isAutoMax) {
+    return 'Min';
+  }
+
+  if (isAutoMax && !isAutoMin) {
+    return 'Max';
+  }
+
+  return 'Auto';
+}
```

Comment:
This used to be implemented with nested ternaries inside `DomainSlider`. Extracting it is how I managed to get rid of the ESLint warning on cognitive complexity. I've kept the function here because it will be removed in a future PR, since the "Auto" toggle will be replaced with an edit button.

Reply:
BTW, when the label of the button changes, the slider moves on the toolbar. That's why I've shortened the labels to "Min" and "Max", so it's closer in length to "Auto". Another good reason to remove this toggle completely.